### DISCUSSION TITLE
ResNetClassifier; fitted object does not seem to be fitted

### DISCUSSION THREAD

**Discussion by cevheck** [alan-turing-institute/sktime#5804] [Q&A]
When using the ResNetClassifier (and the FCNClassifier), I found that my accuracy on my training set was very bad. After setting verbose=True I found out that, even though training appeared very good, the fitted model predicts almost random. See below a toy example.

from sktime.classification.deep_learning import ResNetClassifier, FCNClassifier
import numpy as np

X_train = np.random.rand(10, 1000)
Y_train = np.array([np.sum(X_train[i,:]) > 500 for i in range(X_train.shape[0])])
model = ResNetClassifier(n_epochs=100, batch_size=32, activation="sigmoid", verbose=True)
model.fit(X_train, Y_train)
Y_pred_train = model.predict(X_train)
Y_pred_score = model.score(X_train, Y_train)

print(f"train_score = {sum(Y_pred_train == Y_train)/len(Y_train)}")
print(f'score_function score: {Y_pred_score}')

![image](https://github.com/sktime/sktime/assets/61511837/b40220f2-eab4-4e8a-adf8-b84c66edde1f)

**Comment by cevheck** [comment]
Also created an issue
https://github.com/sktime/sktime/issues/5806

**Reply by fkiraly** [reply]
thx
